Resources/Prototypes/DeltaV/Entities/Clothing/Head/hats.yml

```
@@ -192,3 +192,21 @@
     sprite: DeltaV/Clothing/Head/Hats/beret_corpsman.rsi
   - type: Clothing
     sprite: DeltaV/Clothing/Head/Hats/beret_corpsman.rsi
+
+- type: entity
+  parent: ClothingHeadBase
+  id: ClothingHeadHatStrawHat
+  name: Straw hat
+  description: A fancy hat for hot days, not recommended next to fires.
```

```suggestion
  description: A fancy hat for hot days! Not recommended to wear near fires.
```
I’m on phone so I can’t do a multi line review comment 😭